src/status_im/constants.cljs

```diff
@@ -14,6 +14,7 @@
 (def ^:const content-type-community 9)
 (def ^:const content-type-gap 10)
 (def ^:const content-type-contact-request 11) ;; TODO: temp, will be removed
+(def ^:const content-type-album 12)
```

Comment:
You are adding the same constant in `src/status_im2/common/constants.cljs`, is this needed here?
I would suggest to keep just one constant